src/main/webapp/style.css

```diff
@@ -107,9 +121,33 @@ html {
   width: 50%;
 }
 
-@font-face {
-  font-family: 'Product Sans';
-  font-style: normal;
-  font-weight: 400;
-  src: url('./fonts/product-sans/ProductSansRegular.ttf') format('truetype');
+/* View page */
+
+.halfpage {
+  height: calc(100vh - 56px);
+  max-height: calc(100vh - 56px);
+  max-width: 50vw;
+  width: 50vw;
+  overflow: hidden;
+}
+
+#page {
```

Comment:
Not too sure if this would be here but could you add padding to the top of the page div? The video and comment section seem very close to the nav bar.

Reply:
I'm going to hold off on that for a future PR.  This was mostly the layout of the left/right half-pages.

Reply:
Added to description